src/panels/calendar/dialog-calendar-event-detail.ts

```diff
@@ -122,16 +128,69 @@ class DialogCalendarEventDetail extends LitElement {
   }
 
   private _renderRruleAsText(value: string) {
-    // TODO: Make sure this handles translations
     try {
-      const readableText =
-        value === "" ? "" : RRule.fromString(`RRULE:${value}`).toText();
-      return html`<div id="text">${readableText}</div>`;
+      const rule = RRule.fromString(`RRULE:${value}`);
+      if (rule.isFullyConvertibleToText()) {
+        const readableText =
+          value === ""
+            ? ""
+            : RRule.fromString(`RRULE:${value}`).toText(
+                this._translateRruleElement.bind(this),
```

Comment:
```suggestion
                this._translateRruleElement,
```

Reply:
Let me try that again, but I think without it did not work as then at runtime in the callbacks `this` was invalid.

Reply:
not when it is an arrow function